### PR TITLE
Remove n_islands from config files

### PR DESCRIPTION
**Pull request by rlopezcoto**
As discussed some time ago, with the changes in Data/MC regarding integration windows and correction factors, the number of islands between data and MC was not matched anymore when using the standard cleaning settings, so there was a mismatch in background rates after classification noticed by @moralejo 
The temporary solution is to remove the `n_islands` from the Random Forest training

### CODE REVIEW

File: lstchain/data/lstchain_standard_config.json [hunk 1]
```diff
@@ -68,7 +68,6 @@
     "r",
     "time_gradient",
     "leakage_intensity_width_2",
```
Comment (vuillaut) [inline lstchain/data/lstchain_standard_config.json:70]: last line must not have a `,`

File: lstchain/data/soft_cut_config.json [hunk 2]
```diff
@@ -82,7 +82,6 @@
     "r",
     "time_gradient",
     "leakage_intensity_width_2",
```
Comment (vuillaut) [inline lstchain/data/soft_cut_config.json:84]: same here

Comment (rlopezcoto) [reply lstchain/data/soft_cut_config.json]: done

File: lstchain/data/lstchain_standard_config.json (hunk 1 above)
Comment (rlopezcoto) [reply lstchain/data/lstchain_standard_config.json]: done

**Review by vuillaut — APPROVED** [review]

**Comment by garciagenrique** [comment]
Just for the record (sorry if it seems obvious), I will keep tagging the MC productions with the last released version of lstchain to be consistent. 
Meaning that until we don't have >=v0.6.4, all the prods will contain the `wo_n_islands` tag (until this version is not implemented `n_islands` would still be in the standard config file).

**Comment by rlopezcoto** [comment]
that sounds good to me @garciagenrique

**Pull request merged**